js/cheap.js

```diff
@@ -1,3 +1,16 @@
+window.addEventListener('load', function() {
+  if (typeof web3 !== 'undefined') {
+    console.log('web3 is enabled')
+    if (web3.currentProvider.isMetaMask === true) {
+      console.log('MetaMask is active')
+    } else {
+      alert('MetaMask is not available, please install MetaMask extension')
+    }
+  } else {
+    alert('Web3 is not found. Please install MetaMask extension')
```

Comment:
More a request for clarification for my own knowledge than a comment on the code: is web3 not provided by the CDN?